[BUG] - Table Virtualization Local import code fails in React + TypeScript app while website preview works fine

### HeroUI Version

@heroui/table@2.2.27

### Describe the bug

When using the Table Virtualization example locally (copied directly from the [HeroUI Docs source](https://www.heroui.com/docs/components/table#virtualization) the preview works perfectly fine on the official website of HeroUI. However, when I copy the same code into my local React + TypeScript project, the example fails to render and throws import errors.

It seems the docs example might be importing internal or private utilities that are not accessible through the public @heroui/table package.

### Your Example Website or App

https://github.com/nikhil-304/fix-heroui-issue

### Steps to Reproduce the Bug or Issue

1. Create a new React + TypeScript project.
2. Initialize HeroUI project using the CLI:
`npx heroui init`
`npm install`
3. Add the Table component using the CLI:
`npx heroui add table`
4. Copy the exact code from
https://www.heroui.com/docs/components/table#virtualization
5. Paste it inside App.tsx
6. Run: `npm run dev`
7. Observe browser - the app throws an import error or fails to render.
Module not found: Can't resolve '@nextui-org/react'


### Expected behavior

Copied Table Virtualization examples should work locally using public HeroUI imports (@heroui/table), just like they do on the docs website preview.

### Screenshots or Videos

https://github.com/user-attachments/assets/02a50238-e021-4ec4-8374-4ee2deb7fe3b

### Operating System Version

Windows 11

### Browser

Firefox

--- a/apps/docs/content/components/table/virtualization-custom-max-table-height.raw.jsx
+++ b/apps/docs/content/components/table/virtualization-custom-max-table-height.raw.jsx
@@ -1,4 +1,4 @@
-import {Table, TableBody, TableCell, TableColumn, TableHeader, TableRow} from "@nextui-org/react";
+import {Table, TableBody, TableCell, TableColumn, TableHeader, TableRow} from "@heroui/react";
 
 function generateRows(count) {
   return Array.from({length: count}, (_, index) => ({

--- a/apps/docs/content/components/table/virtualization-custom-row-height.raw.jsx
+++ b/apps/docs/content/components/table/virtualization-custom-row-height.raw.jsx
@@ -1,4 +1,4 @@
-import {Table, TableBody, TableCell, TableColumn, TableHeader, TableRow} from "@nextui-org/react";
+import {Table, TableBody, TableCell, TableColumn, TableHeader, TableRow} from "@heroui/react";
 
 function generateRows(count) {
   return Array.from({length: count}, (_, index) => ({

--- a/apps/docs/content/components/table/virtualization-ten-thousand.raw.jsx
+++ b/apps/docs/content/components/table/virtualization-ten-thousand.raw.jsx
@@ -1,4 +1,4 @@
-import {Table, TableBody, TableCell, TableColumn, TableHeader, TableRow} from "@nextui-org/react";
+import {Table, TableBody, TableCell, TableColumn, TableHeader, TableRow} from "@heroui/react";
 
 function generateRows(count) {
   return Array.from({length: count}, (_, index) => ({

--- a/apps/docs/content/components/table/virtualization.raw.jsx
+++ b/apps/docs/content/components/table/virtualization.raw.jsx
@@ -1,4 +1,4 @@
-import {Table, TableBody, TableCell, TableColumn, TableHeader, TableRow} from "@nextui-org/react";
+import {Table, TableBody, TableCell, TableColumn, TableHeader, TableRow} from "@heroui/react";
 
 function generateRows(count) {
   return Array.from({length: count}, (_, index) => ({

